packages/fast-usdc/src/exos/advancer.js

```diff
@@ -116,6 +116,7 @@ export const prepareAdvancerKit = (
      *   notifyFacet: import('./settler.js').SettlerKit['notify'];
      *   borrowerFacet: LiquidityPoolKit['borrower'];
      *   poolAccount: HostInterface<OrchestrationAccount<{chainId: 'agoric'}>>;
+     *   intermediateRecipient: ChainAddress;
```

Comment:
we seem to throw away all but the `.value` without checking it. Just use a string?

Reply:
better to check it, but even without checking it there's utility in knowing it's not just a string but a ChainAddress